rust-src/concordium_base/src/web3id/mod.rs

```diff
@@ -1000,6 +1000,65 @@ pub enum CommitmentInputs<'a, C: Curve, AttributeType, Web3IdSigner> {
     },
 }
 
+#[serde_with::serde_as]
+#[derive(serde::Deserialize)]
+#[serde(bound(deserialize = "AttributeType: DeserializeOwned, Web3IdSigner: DeserializeOwned"))]
+#[serde(rename_all = "camelCase", tag = "type")]
```

Comment:
Does this rename `Account` to `account` as well?

So we have
```
"type": "account"
```
?

Reply:
Yes, this renames only the tags. (Account/Web3Issuer)